youtube_dl/extractor/arte.py

```diff
@@ -199,25 +199,16 @@ class ArteTVCreativeIE(ArteTVPlus7IE):
 
 class ArteTVFutureIE(ArteTVPlus7IE):
     IE_NAME = 'arte.tv:future'
-    _VALID_URL = r'https?://future\.arte\.tv/(?P<lang>fr|de)/(thema|sujet)/.*?#article-anchor-(?P<id>\d+)'
+    _VALID_URL = r'https?://future\.arte\.tv/(?P<lang>fr|de)/[^/]+/(?P<id>.+)'
```

Comment:
Still to restrictive for http://future.arte.tv/fr/la-science-est-elle-responsable.